drydock_provisioner/objects/task.py

```diff
@@ -109,17 +102,14 @@ class DriverTask(Task):
     def __init__(self, task_scope={}, **kwargs):
         super(DriverTask, self).__init__(**kwargs)
 
-        self.design_id = kwargs.get('design_id', 0)
-
-        self.site_name = task_scope.get('site', None)
+        self.design_id = kwargs.get('design_id', None)
```

Comment:
I believe the default value for dict.get()  is None if the key isn't present: https://www.tutorialspoint.com/python/dictionary_get.htm